app/services/fh_services.rb

```diff
@@ -11,15 +11,19 @@ def get_company
     @connection.get "?api-app="+"#{ENV["APPKEY"]}"+"&api-user="+"#{ENV["user_key"]}"
   end
 
-  def get_item
-    @connection.get "marina/items/?api-app=" + "#{ENV["APPKEY"]}"+"&api-user="+"#{ENV["user_key"]}"
+  def get_item(company)
+    if company.class == String
```

Comment:
I would prefer `company.is_a?(String)`. The reason is that `#is_a?` checks whether the class or one of the superclasses of the object your inspecting is a `String`. Seems trivial, but every once in a while your String will be wrapped in an object, kind of like how ActiveRecordRelation behaves like an array, but is actually an entirely different object.